.gitignore

```diff
@@ -45,3 +45,5 @@ nlohmann_json.spdx
 
 # Bazel-related
 MODULE.bazel.lock
+enum.log
+tidy.log
```

Comment:
Why did you edit .gitignore?

Reply:
The last successful CI run (https://github.com/nlohmann/json/actions/runs/16006669329/job/45154866131?pr=4587) is already using Clang-Tidy 21.0.0.

Reply:
I'm already addressing these in #4871.

Reply:
I think I get the Ubuntu stuff to work soon. For Windows, I could use some help. Best case scenario would be to keep all existing jobs running, but I guess that some are bound to the deprecated/removed images. It would be great if you could have a look.

Best start from the develop branch (not #4871 where I already deleted some jobs) and just ignore the errors in Ubuntu for the time being.

Reply:
I created #4875 for this. It would be great if you could take a look!